src/main/java/io/dashboardhub/pipelinedashboard/controller/AccountController.java

```diff
@@ -0,0 +1,24 @@
+package io.dashboardhub.pipelinedashboard.controller;
+
+import io.dashboardhub.pipelinedashboard.domain.User;
+import io.dashboardhub.pipelinedashboard.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class AccountController {
+
+    @Autowired
+    private UserService userService;
+
+    @RequestMapping(value = {"/profile"}, method = RequestMethod.GET)
+    public String profile(Model model) {
+        User user = this.userService.findByCurrentUsername();
```

Comment:
Ok, I will remove the `this` where not required - which is probably everywhere :smile: